jax/experimental/jax2tf/jax2tf.py

```diff
@@ -809,6 +810,79 @@ def _argminmax(fn, operand, axes, index_dtype):
 tf_impl[lax.cumsum_p] = tf.math.cumsum
 tf_impl[lax.cumprod_p] = tf.math.cumprod
 
+def _select_and_gather_add(tangents: TfVal,
+                           operand: TfVal,
+                           select_prim: core.Primitive,
+                           window_dimensions: Sequence[int],
+                           window_strides: Sequence[int],
+                           base_dilation: Sequence[int],
+                           window_dilation: Sequence[int],
+                           padding: Sequence[Tuple[int, int]],
+                           max_bits=64):
```

Comment:
There is a problem here, because max_bits will always be 64, even on TPUs. jax2tf does not have a mechanism to use a separate implementation for TPUs.

I think that a solution might be to always use 64 bits for max_bits, and to let XLA throw the error that it cannot handle it, on TPU.

Reply:
That means we can then leave the parameter as such? I think it would make the most sense anyway – seeing as it is set like this in `lax._select_and_gather_add_translation`.

Reply:
After offline discussion, dropped the `max_bits` parameter in `7709c1ae` and specialized for 64 bits.